frontend/assets/stylesheets/components/_bundle-header.scss

```diff
@@ -0,0 +1,31 @@
+
+/* Header */
+/* Green Part of the Header */
+.bundle-header--main-header {
+    background: #87be27;
+}
+
+/* White Part of the Header */
+.bundle-header--secondary-header {
+
+}
+
+.bundle-a__title {
+    @include text-title();
+    padding-top: 24px;
+}
+
+.bundle-a__tagline {
+    @include text-tag-line();
+    padding-bottom: 36px;
+    width: 460px;
+}
+
+.bundle-header__img {
+    position: absolute;
+}
+
+.bundle-header__paragraph{
+    width: 460px;
```

Comment:
If possible, it is preferable to use [`gs-span($n-columns)`](https://github.com/guardian/membership-frontend/blob/master/frontend/assets/stylesheets/_mixins.scss#L24) to represent element widths